Azure/communication-ui-library-android
# Features/hang up on call

## Purpose

* Hang up on call accepted

## Does this introduce a breaking change?

```
[] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test


* On clean install (needs initial permissions prompt)
* Join a call
* Receive and accept a PSTN call, observe Composite hang up.


## What to Check
Verify that the following are valid
* ...

## Other Information
